examples/sv/gnomAD_DUP_chr1:10000-20000.json

```diff
@@ -0,0 +1,66 @@
+{   
+   "name": "gnomAD_DUP_chr1:10000-20000.json",
```

Comment:
This example was picked from CADD file for SV, are we currently only handling variants with an accession?